config/initializers/sidekiq.rb

```diff
@@ -4,6 +4,9 @@
 Sidekiq::Logging.logger.level = Logger::WARN
 Sidekiq::Logging.logger.formatter = SidekiqLoggerFormatter.new
 
+# for non-production logging, use:
+# Rails.logger = Sidekiq::Logging.logger
```

Comment:
If this were not commented out, this would overwrite the logger for the web process as well?